.github/xoops-copilot-template.md

```
@@ -0,0 +1,102 @@
+composer fix              # Auto-fix code style issues
+```
+
+## PHP Compatibility
+
+Code must run on PHP 7.4 through 8.5. Do not use features exclusive to PHP 8.0+ (named arguments, match expressions, union type hints in signatures, enums, fibers, readonly properties, intersection types, `never` return type, first-class callable syntax).
+
+## XOOPS Coding Conventions
+
+- Follow PSR-12 coding standard.
+- Every source file begins with the XOOPS copyright header block:
+  ```php
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)